exercises/word-search/src/test/java/WordSearcherTest.java

```diff
@@ -0,0 +1,260 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.junit.Assert.assertEquals;
+
+/*
+ * version: 1.0.0
+ */
+public class WordSearcherTest {
+
+    private WordSearcher wordSearcher;
+
+    @Before
+    public void setUp() {
+        wordSearcher = new WordSearcher();
+    }
+
+    @Test
+    public void testLocatesWordsWrittenLeftToRight() {
+        Map<String, Optional<WordLocation>> expectedLocations = new HashMap<>();
+        expectedLocations.put("clojure", Optional.of(new WordLocation(new Pair(1, 10), new Pair(7, 10))));
+
+        final Map<String, Optional<WordLocation>> actualLocations = wordSearcher.search(
+                expectedLocations.keySet(),
```

Comment:
I think keeping it like this is okay :) It might be a bit clearer to declare the set separately and name it `wordsToSearchFor` or something. But it's fine as it is :)